index.bs

```diff
@@ -10970,14 +10977,14 @@ then the <emu-val>Function</emu-val> object is [=%ArrayProto_values%=].
         1.  Return [=CreateSetIterator=](|backing|, <code>"value"</code>).
 </div>
 
-The value of the [=@@iterator=] <emu-val>Function</emu-val> object’s “length”
-property is the <emu-val>Number</emu-val> value <emu-val>0</emu-val>.
+The value of the [=@@iterator=] [=function object=]’s “length”
+property is the Number value <emu-val>0</emu-val>.
 
-The value of the [=@@iterator=] <emu-val>Function</emu-val> object’s “name” property
-is the <emu-val>String</emu-val> value “entries”
+The value of the [=@@iterator=] [=function object=]’s “name” property
+is the String value “entries”
 if the interface has a [=pair iterator=] or a [=maplike declaration=]
-and the <emu-val>String</emu-val> “values”
-if the interface has a [=setlike declaration=].
+and the String “values”
+if the interface has a [=setlike declaration=] or defines an [=indexed property getter=].
```

Comment:
This seem like an orthogonal issue that would be better left in a separate PR, no?